### PR TITLE
Add options to hide icon, show speeds in bits or bytes, and show upload and download at the same time

### PR DESCRIPTION
**Pull request by stringlapse**
The settings to show the icon and use bytes are set by default to work the same as the previous behavior. I didn't update the screenshot in the readme because I only have one network interface so I can't show how multiple interfaces are supported. I saw showing upload and download at the same time was in TODO.md so I marked it off there.

### CODE REVIEW

File: utils/GenUtils.js [hunk 1]
```diff
@@ -60,10 +61,10 @@ function compareJsonStrings(str1, str2) {
  * @param {number} mode  0 - bits mode,  1 - bytes mode
  * @returns - string representation of data speed
  */
-function bytesSpeedToString(amount, mode = 1) {
+function bytesSpeedToString(amount, mode = appSettingsModel.displayBytes) {
```
Comment (noroadsleft000) [inline utils/GenUtils.js:64]: Can you pass some hard coded value true.
This is reusable code, Idea is not to add model dependency on Utils.

File: prefs.js [hunk 2]
```diff
@@ -308,6 +314,38 @@ class PrefsApp {
         this.schema.bind(SettingKeys.RESET_MINUTES, this._resetMinutesInput, 'value', Gio.SettingsBindFlags.DEFAULT);
     }
 
+    // 7. Show numbers in bytes instead of bits
+    _createUnitToggleControl() {
+        const unitLabel = new Gtk.Label({
+            label: _(_("Show speeds in bytes instead of bits")),
```
Comment (noroadsleft000) [inline prefs.js:320]: you mistakenly called _() twice.

File: prefs.js [hunk 3]
```diff
@@ -308,6 +314,38 @@ class PrefsApp {
...
+            visible: true
+        });
+        this._addRow(unitLabel, this._unitSwitch, SettingRowOrder.DISPLAY_BYTES);
+        this.schema.bind(SettingKeys.DISPLAY_BYTES, this._unitSwitch, 'state', Gio.SettingsBindFlags.DEFAULT);
+    }
+
+    // 8. Show icon in status bar
+    _createIconToggleControl() {
+        const iconLabel = new Gtk.Label({
+            label: _(_("Show icon in status bar (requires reload)")),
```
Comment (noroadsleft000) [inline prefs.js:336]: you mistakenly called _() twice.

File: utils/GenUtils.js [hunk 4]
```diff
@@ -60,10 +61,10 @@ function compareJsonStrings(str1, str2) {
  * @param {number} mode  0 - bits mode,  1 - bytes mode
```
Comment (noroadsleft000) [inline utils/GenUtils.js:61]: Please change the type from` number -> boolean` in comments.

**Review by noroadsleft000 — CHANGES_REQUESTED** [review]

**Comment by noroadsleft000** [comment]
Seeing no activity on this for long time.
And, since there are some significant features added by the contributor.
We will go ahead with merging and address review comments in separate PR.

**Comment by stringlapse** [comment]
Apologies, life got really busy for me. Thank you for fixing the problems and merging!

**Pull request merged**